File: Q2/Q2.c
/*Find the largest of three numbers.*/
#include<stdio.h>
void main()
{
    int  a, b, c, largest;

    printf("Enter First Number : ");
    scanf("%d" , &a);

    printf("Enter Second Number : ");
    scanf("%d" , &b);

    printf("Enter Third Number : ");
    scanf("%d" , &c);

    if(a>b && a>c)
    {
        printf("Largest Number is : %d\n" , a);
    }

    else if(b>a && b>c)
    {
        printf("Largest Number is : %d\n", b);
    }

    else
    {
        printf("Largest Number is : %d\n", c);
    }

}

File: Q2/Q2.go
package main

import "fmt"

func main() {
    var a, b, c int

    fmt.Print("Enter First Number: ")
    fmt.Scan(&a)

    fmt.Print("Enter Second Number: ")
    fmt.Scan(&b)

    fmt.Print("Enter Third Number: ")
    fmt.Scan(&c)

    if a > b && a > c {
        fmt.Printf("Largest Number is: %d\n", a)
    } else if b > a && b > c {
        fmt.Printf("Largest Number is: %d\n", b)
    } else {
        fmt.Printf("Largest Number is: %d\n", c)
    }
}
